### DISCUSSION TITLE
Firefighting event trash can doesn't "smoke" in Multiplayer

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#14008] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Firefighting event trash can is supposed to be smoking, but doesn't in MP, if the event appears naturally
It smokes just fine if triggered by console

Part of larger series of statuseffect MP desyincs

![image](https://github.com/FakeFishGames/Barotrauma/assets/73229309/71620072-1a10-4a43-b680-3f47c6b1105e)

### Reproduction steps

1. increase event commonness to 100000
2. go to area with difficulty > 30 
3. wait until it triggers by itself

### Bug prevalence

Just once

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Duplicate of https://github.com/FakeFishGames/Barotrauma/issues/13406
